test/parallel/test-fs-read-stream-file-handle.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+const common = require('../common');
+const fs = require('fs');
+const assert = require('assert');
+const path = require('path');
+const tmpdir = require('../common/tmpdir');
+const file = path.join(tmpdir.path, '/read_stream_filehandle_test.txt');
```

Comment:
To fix the error on Windows CI:
```suggestion
const file = path.join(tmpdir.path, 'read_stream_filehandle_test.txt');
```